From cae2b341c35f35fd7bff7c9809c348cd0be5143e Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Fri, 7 Jun 2019 08:56:40 -0700
Subject: [PATCH 1/2] Add stateless support

---
 sanei/sanei_config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sanei/sanei_config.c b/sanei/sanei_config.c
index c158766..ae901e9 100644
--- a/sanei/sanei_config.c
+++ b/sanei/sanei_config.c
@@ -67,7 +67,7 @@
 # define PATH_SEP	'/'
 #endif
 
-#define DEFAULT_DIRS	"." DIR_SEP STRINGIFY(PATH_SANE_CONFIG_DIR)
+#define DEFAULT_DIRS	"." DIR_SEP STRINGIFY(PATH_SANE_CONFIG_DIR) DIR_SEP "/usr/share/defaults/sane"
 
 #ifdef __BEOS__
 #include <FindDirectory.h>
-- 
2.23.0

